x/mint/keeper/migrator.go

```diff
@@ -21,3 +25,23 @@ func NewMigrator(k Keeper) Migrator {
 func (m Migrator) Migrate1to2(ctx context.Context) error {
 	return nil
 }
+
+// Migrate2to3 migrates the x/mint module state from the consensus version 2 to
+// version 3.
+func (m Migrator) Migrate2to3(ctx context.Context) error {
+	params, err := m.keeper.Params.Get(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Initialize the new MaxSupply parameter with the default value
+	params.MaxSupply = math.ZeroInt()
```

Comment:
nit, for consistency with the other migrations, get the default config and set it as defaultConfig.MaxSupply